buildenv/jenkins/common/build.groovy

```diff
@@ -500,6 +500,16 @@ def fetchFile(src, dest) {
 def archive_diagnostics() {
     def datestamp = variableFile.get_date()
     def diagnosticsFilename = "${JOB_NAME}-${BUILD_NUMBER}-${datestamp}-diagnostics.tar.gz"
+
+    def archiveCmd = "tar -zcvf ${diagnosticsFilename} -T -"
+
+    def findNameCore = "-name 'core.*.dmp'"
+    def findCmd = "find ."
+    findCmd += " ${findNameCore}"
```

Comment:
Suggest inserting "find ." into `findNameCore`:
```sh
    def findNameCore = "find . -name 'core.*.dmp'"
    def findCmd = "${findNameCore}"
```
and fixing the other use of `${findNameCore}`.

Reply:
Done - this is now `findCrashDataCmd`